vpd-manager/manager.cpp

```diff
@@ -251,6 +251,25 @@ LocationCode Manager::getExpandedLocationCode(const LocationCode locationCode,
                                         fruLocationCode);
 }
 
+static auto isPrePostActionRequired(const nlohmann::json& json,
```

Comment:
This function name is not accurate, it isn't just checking, it is actually executing the pre action

Reply:
I disagree, Yes, it is executing the pre action after checking if it is required, that is all. Please document the return codes in the doxygen and rename this function. In fact, I am not even seeing the need for this wrapper function. The return codes are obscure and confusing. Why not just have the code inline?

Reply:
a) If you agree that it executes preAction only after checking and sets the parameter to detect if postAction is required, then why there is a disagreement with function name? 
b) I can remove this wrapper function but then we will have duplicate code at line 285 and 343 which will be exactly the body of this wrapper function. If that is ok pls. let me know, in that case point "a" is irrelevant.

Reply:
> a) If you agree that it executes preAction only after checking and sets the parameter to detect if postAction is required, then why there is a disagreement with function name? b) I can remove this wrapper function but then we will have duplicate code at line 285 and 343 which will be exactly the body of this wrapper function. If that is ok pls. let me know, in that case point "a" is irrelevant.

Because it is trying to convey too many things and using magic return codes to do so.

Yes, please either remove this wrapper or document it well.